src/main/kotlin/com/sourcegraph/cody/edit/EditCommandPrompt.kt

```diff
@@ -48,7 +48,7 @@ class EditCommandPrompt(val controller: FixupService, val editor: Editor, val di
       LlmDropdown(
           modelUsage = ModelUsage.EDIT,
           project = controller.project,
-          onSetSelectedItem = controller::setCurrentModel,
+          onSetSelectedItem = {},
```

Comment:
this callback should persist the default for the inline edits, right?

Reply:
No, @danielmarquespt said the default between edits and chats should be the same.
We are already saving the default in `LlmDropdown::setSelectedItem`